core/expressions.py

```diff
@@ -547,6 +572,19 @@ def __str__(self):
             elif self.value == 10:
                 return "not in"
 
+    Operator.REVERSE_OPERATOR = {
```

Comment:
Why is this here and not inside the subclass Operator? This causes an unresolved attribute reference warning in reverse_operator.